examples/hoomd-blue/Butane_ANN_OpenMM.md

```diff
@@ -0,0 +1,380 @@
+        [1, 7], [2, 7], [3, 7],
+        [11, 4], [12, 4], [13, 4],
+        # HCCH
+        [1, 5], [1, 6], [2, 5], [2, 6], [3, 5], [3, 6],
+        [5, 8], [6, 8], [5, 9], [6, 9],
+        [8, 11], [8, 12], [8, 13], [9, 11], [9, 12], [9, 13]
+    ]
+
+    hoomd.init.read_snapshot(snapshot)
+
+    ### Set interactions
+    nl_ex = hoomd.md.nlist.cell()
+    nl_ex.reset_exclusions(exclusions = ["1-2", "1-3", "1-4"])
+
+    lj = hoomd.md.pair.lj(r_cut = 12.0, nlist = nl_ex)
+    lj.pair_coeff.set('C', 'C', epsilon = 0.07, sigma = 3.55)
+    lj.pair_coeff.set('H', 'H', epsilon = 0.03, sigma = 2.42)
+    lj.pair_coeff.set('C', 'H', epsilon = numpy.sqrt(0.07*0.03), sigma = numpy.sqrt(3.55*2.42))
+
+    coulomb = hoomd.md.charge.pppm(hoomd.group.charged(), nlist = nl_ex)
+    coulomb.set_params(Nx = 64, Ny = 64, Nz = 64, order = 6, rcut = 12.0)
+
+    harmonic = hoomd.md.bond.harmonic()
+    harmonic.bond_coeff.set("CC", k = 2*268.0, r0 = 1.529)
+    harmonic.bond_coeff.set("CH", k = 2*340.0, r0 = 1.09)
+
+    angle = hoomd.md.angle.harmonic()
+    angle.angle_coeff.set("CCC", k = 2*58.35, t0 = 112.7 * pi / 180)
+    angle.angle_coeff.set("CCH", k = 2*37.5, t0 = 110.7 * pi / 180)
+    angle.angle_coeff.set("HCH", k = 2*33.0, t0 = 107.8 * pi / 180)
+
+
+    dihedral = hoomd.md.dihedral.opls()
+    dihedral.dihedral_coeff.set("CCCC", k1 = 1.3, k2 = -0.05, k3 = 0.2, k4 = 0.0)
+    dihedral.dihedral_coeff.set("HCCC", k1 = 0.0, k2 = 0.0, k3 = 0.3, k4 = 0.0)
+    dihedral.dihedral_coeff.set("HCCH", k1 = 0.0, k2 = 0.0, k3 = 0.3, k4 = 0.0)
+
+    lj_special_pairs = hoomd.md.special_pair.lj()
+    lj_special_pairs.pair_coeff.set("CCCC", epsilon = 0.07, sigma = 3.55, r_cut = 12.0)
+    lj_special_pairs.pair_coeff.set("HCCH", epsilon = 0.03, sigma = 2.42, r_cut = 12.0)
+    lj_special_pairs.pair_coeff.set("HCCC",
+        epsilon = numpy.sqrt(0.07 * 0.03), sigma = numpy.sqrt(3.55 * 2.42), r_cut = 12.0
+    )
+
+    coulomb_special_pairs = hoomd.md.special_pair.coulomb()
+    coulomb_special_pairs.pair_coeff.set("CCCC", alpha = 0.5, r_cut = 12.0)
+    coulomb_special_pairs.pair_coeff.set("HCCC", alpha = 0.5, r_cut = 12.0)
+    coulomb_special_pairs.pair_coeff.set("HCCH", alpha = 0.5, r_cut = 12.0)
+
+    hoomd.md.integrate.mode_standard(dt = dt)
+    integrator = hoomd.md.integrate.nvt(group = hoomd.group.all(), kT = kT, tau = 100*dt)
+    integrator.randomize_velocities(seed = 42)
+
+    return hoomd.context.current
+```
+
+<!-- #region id="3UrzENm_oo6U" -->
+Next we load PySAGES and the relevant classes and methods for our problem
+<!-- #endregion -->
+
+```python id="fpMg-o8WomAA"
+from pysages.grids import Grid
+from pysages.collective_variables import DihedralAngle
+from pysages.methods import ANN
+
+import pysages
+```
+
+<!-- #region id="LknkRvo1o4av" -->
+The next step is to define the collective variable (CV). In this case, we choose the central dihedral angle.
+
+We also define a grid to bin our CV space, and a topology for our neural
```

Comment:
Since this is more or less the documentation, you need to expand on this.
How do you choose the grid? What are the parameter and why, how would it look like for more than 1D?

What is the topology?
I think there is an opportunity for expanding on this here. If you have the explanation somewhere else, you can link to it though.